### PR TITLE
Solution cleanup

### PR DESCRIPTION
**Pull request by CaptnCodr**
@sergey-tihon I request a review in case I overlooked something.
Thank you.

### CODE REVIEW

File: src/FsUnit.NUnit/CustomConstraints.fs [hunk 1]
```diff
@@ -17,6 +17,4 @@ module CustomConstraints =
                 let result = Common.isOfCase this.Expected actual
                 ConstraintResult(this, actual, result)
             else
-                let actualType = actual.GetType()
-                do printf $"Got a {actualType.Name}"
```
Comment (sergey-tihon) [inline src/FsUnit.NUnit/CustomConstraints.fs:21]: Should not it be inside the error message?

File: src/FsUnit.Xunit/CustomMatchers.fs [hunk 2]
```diff
@@ -185,9 +185,9 @@ let instanceOfType<'a> =
 let contain expected =
     let matches(actual: obj) =
         match actual with
-        | :? (list<_>) as l -> l |> List.exists(fun i -> i = expected)
-        | :? (array<_>) as a -> a |> Array.exists(fun i -> i = expected)
-        | :? (seq<_>) as s -> s |> Seq.exists(fun i -> i = expected)
+        | :? list<_> as l -> l |> List.exists(fun i -> i = expected)
```
Comment (sergey-tihon) [inline src/FsUnit.Xunit/CustomMatchers.fs:188]: do you like syntax
```fsharp
List.exists (=) expected
```
? 🙈

File: tests/FsUnit.Xunit.Test/typed.shouldHaveLengthTests.fs [hunk 3]
```diff
@@ -3,7 +3,8 @@ namespace FsUnit.Typed.Test
 open Xunit
 open FsUnitTyped
 
-type ``haveLength tests``() =
+type ``shouldHaveLength tests``() =
+
```
Comment (sergey-tihon) [inline tests/FsUnit.Xunit.Test/typed.shouldHaveLengthTests.fs:7]: all files have this new line after type, is it added by Fantomas? If yes, I do not see Fantomas config changes in this PR...

**Review by sergey-tihon — COMMENTED** [review]
Thank you for this PR, looks good in general. Just have few questions.

File: tests/FsUnit.Xunit.Test/typed.shouldHaveLengthTests.fs (hunk 3 above)
Comment (CaptnCodr) [reply tests/FsUnit.Xunit.Test/typed.shouldHaveLengthTests.fs:7]: I added that manually for better readability.

File: src/FsUnit.Xunit/CustomMatchers.fs (hunk 2 above)
Comment (CaptnCodr) [reply src/FsUnit.Xunit/CustomMatchers.fs:188]: That's a good suggestion. I'll apply that.

File: src/FsUnit.NUnit/CustomConstraints.fs (hunk 1 above)
Comment (CaptnCodr) [reply src/FsUnit.NUnit/CustomConstraints.fs:21]: Gotcha.

**Pull request merged**